apps/external-devices/device-descriptors/5DT_Data_Glove_14_Utra.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Fifth Dimension Technologies",
+    "deviceName": "5DT Data Glove 14 Utra",
```

Comment:
@rpavlik Thanks! I'll fix this in the next batch.

Right now I'm skipping the mappings. The primary goal at the moment is to flesh out the supported devices table on the website.